thirtyinch/src/main/java/net/grandcentrix/thirtyinch/serialize/TiPresenterSerializer.java

```diff
@@ -0,0 +1,40 @@
+package net.grandcentrix.thirtyinch.serialize;
+
+import android.support.annotation.NonNull;
+
+import net.grandcentrix.thirtyinch.TiPresenter;
+import net.grandcentrix.thirtyinch.TiView;
+
+/**
+ * Interface to serialize and deserialize presenters. The interface should be implemented, if
+ * a {@link TiPresenter} should be restored, after a process has died, but the UI component is
+ * being recreated.
+ */
+public interface TiPresenterSerializer {
```

Comment:
Dang, I just realized that it's not necessary to make any changes in the config class, because in order to access the config, you need to create a presenter first. 

There's no way to replace the savior in the delegate from `TiFragment` or `TiActivity` at the moment. I think it's better to add a new provider interface for the savior and the default implementation delegate to the `PresenterSavior`. Although a global config would be nicer.